checker/tests/nullness/java8/methodref/ClassTypeArgInference.java

```diff
@@ -0,0 +1,50 @@
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ClassTypeArgInference {
+    public static void main(String[] args) {
+        Gen<String> o = new Gen<>("");
+        // :: error: (methodref.param.invalid)
+        Factory f = Gen::make;
+        // :: error: (methodref.param.invalid)
+        Factory f2 = Gen<String>::make;
+        // :: error: (methodref.receiver.invalid) :: error: (methodref.return.invalid)
+        Factory f3 = Gen<@Nullable String>::make;
+        f2.make(o, null).toString();
+
+        /*  Factory2 factory2a = Gen<@Nullable String>::new;
```

Comment:
What do these comments mean? Are these more tests that should be enabled?
Class Factory2 is currently unused.